soh/soh/Enhancements/randomizer/randomizer_item_tracker.cpp

```diff
@@ -640,11 +682,20 @@ void UpdateVectors() {
 
     dungeonItems.clear();
     if (CVar_GetS32("gItemTrackerDisplayDungeonItemsHorizontal", 1) && CVar_GetS32("gItemTrackerDungeonItemsDisplayType", 2) == 2) {
-        dungeonItems = GetDungeonItemsVector(itemTrackerDungeons, 12);
-        dungeonItems[23] = ITEM_TRACKER_ITEM(ITEM_KEY_SMALL, SCENE_GERUDOWAY, DrawDungeonItem);
+        if (CVar_GetS32("gItemTrackerDisplayDungeonItemsMaps", 1)) {
+            dungeonItems = GetDungeonItemsVector(itemTrackerDungeonsWithMapsHorizontal, 12);
+            dungeonItems[23] = ITEM_TRACKER_ITEM(ITEM_KEY_SMALL, SCENE_GERUDOWAY, DrawDungeonItem);
```

Comment:
i guess i'm not sure what the "manually for space" comment meant...

Reply:
So the gist is we’re iterating over the vectors to display the items in rows and columns, in the case of dungeons it’s awkward because we have 6 columns, and 13 dungeons, so there’s ends up being an entire additional row for the last dungeon.

So the quick fix for this is just don’t include the last dungeon in the vector, and place it by hand exactly where you want it, in this case it happens to be right below the GTG key because there is an empty space there

Reply:
coolcool, is there a plan to eventually use the entry in `itemTrackerDungeonsWithMapsHorizontal`, if not, would you mind removing the commented out entries and adding a little comment next to the spots where the icons are manually placed explaining what's happening?

Reply:
Yep can do